src/styles/header/_toolbar.scss

```diff
@@ -3,139 +3,181 @@
-      content: $fa-var-external-link;
-    }
+      &[title='Generate Snapshot']::before {
+        content: $fa-var-code-fork;
+      }
 
-    &[title='Edit Combine Script']::before {
-      content: $fa-var-gears;
-    }
+      &[title='Sort By …']::before {
+        content: $fa-var-sort;
+      }
 
-    &[title='Filter']::before {
-      content: $fa-var-filter;
-    }
+      &[title='Visualization …']::before {
+        content: $fa-var-exchange;
+      }
 
-    &[title='Stratify By']::before,
-    &[title='Stratify By Threshold']::before {
-      content: $fa-var-window-maximize;
-    }
+      &[title='Edit Link Pattern …']::before {
+        content: $fa-var-external-link;
+      }
 
-    &[title='Set Cut Off']::before {
-      content: $fa-var-scissors;
-    }
+      &[title='Edit Combine Script …']::before {
+        content: $fa-var-gears;
+      }
 
-    &[title='Search']::before {
-      content: $fa-var-search;
-    }
+      &[title='Filter …']::before {
+        content: $fa-var-filter;
+      }
 
-    &[title='Edit Weights']::before {
-      content: $fa-var-tasks;
-    }
+      &[title='Stratify']::before,
+      &[title='Stratify By Threshold …']::before {
+        content: $fa-var-bars;
+      }
 
-    &[title='UnCollapse']::before {
-      content: $fa-var-toggle-right;
-    }
+      &[title='Stratify'][data-stratify='true']::before,
+      &[title='Stratify By Threshold …'][data-stratify='true']::before {
+        color: $lu_selected_color;
+      }
 
-    &[title='Collapse']::before {
-      content: $fa-var-toggle-left;
-    }
+      &[title='Set Cut Off …']::before {
+        content: $fa-var-scissors;
+      }
 
-    &[title='Expand']::before {
-      content: $fa-var-expand;
-    }
+      &[title='Search …']::before {
+        content: $fa-var-search;
+      }
 
-    &[title='Compress']::before {
-      content: $fa-var-compress;
-    }
+      &[title='Edit Weights …']::before {
+        content: $fa-var-tasks;
+      }
+
+      &[title='UnCollapse']::before {
+        content: $fa-var-toggle-right;
+      }
+
+      &[title='Collapse']::before {
+        content: $fa-var-toggle-left;
+      }
 
-    &[title='Hide']::before {
-      content: $fa-var-times;
+      &[title='Expand']::before {
+        content: $fa-var-expand;
+      }
+
+      &[title='Compress']::before {
+        content: $fa-var-compress;
+      }
+
+      &[title='Remove']::before {
+        content: $fa-var-trash;
+      }
+
+      &[title='More …']::before {
+        content: $fa-var-ellipsis-h;
+      }
+
+      &[title='Sort']::before,
+      &[title='Sort'][data-sort='asc']::before {
+        content: $fa-var-sort-amount-asc;
+      }
+
+      &[title='Sort'][data-sort='desc']::before {
+        content: $fa-var-sort-amount-desc;
+      }
     }
   }
-}
 
-&:not(.#{$lu_css_prefix}-compressed):hover {
-  .#{$lu_css_prefix}-toolbar {
-    display: block;
-    padding: 7px;
+  /* Override alphabetical sort icons */
+  .categorical,
+  .string,
```

Comment:
what about: link, annotate, hierarchy

Reply:
Github doesn't get the change. But I addressed it here: https://github.com/Caleydo/lineupjs/pull/333/commits/3ea13efd72da2bcc069839fd37c503f2d85fbf6d